### ISSUE TITLE
carla-bridge-native vst2: enable send control changes option by command line

### ISSUE THREAD

**Issue by ohac** [falkTX/Carla#301]
I wanted to enable PLUGIN_OPTION_SEND_CONTROL_CHANGES by command line.
But I couldn't find such command line option.
So I patched to CarlaPluginVST2.cpp.

https://gist.github.com/ohac/348cc4069f6c4f4c78ee

Now I can use modulation wheel and sustain pedal.
Are there any command line option?

I use this command:

```
carla-git-1.9.5~git20150817/bin/carla-bridge-native vst2 /path/to/M1.so '(none)' 0
```

(M1.so is created by Airwave. M1 is KORG's VST.)

**Comment by falkTX** [comment]
there's command-line option for this, but you can load the plugin in carla (gui) and save a preset.
then with after carla-single is running, send a SIGUSR1 signal to it to trigger a file save.
replace that file with the presets contents, and next time you run carla-single it will load that preset (by loading the replaced file)

PS: the plugin options thing is a bitmask.

alternatively I can add a environment variable that forces the options, would that work for you?

**Comment by ohac** [comment]
Thanks for reply.
I found ~/.config/falkTX/CarlaPlugins2.conf. Is this the preset?
I prefer a environment variable. Because .conf is too hard to edit without GUI.

**Comment by falkTX** [comment]
Fixed in 6d176142cc07a81bc8677fa83a5e3c37d0e10605
CCs will always be enabled when starting the bridges manually or via carla-single.
